lib/core/CLoggerThrottler.cc

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#include <core/CLoggerThrottler.h>
+
+#include <core/CTimeUtils.h>
+
+#include <limits>
+#include <iostream>
```

Comment:
Not needed?

Reply:
See [65caeb0](https://github.com/elastic/ml-cpp/pull/1615/commits/65caeb0d5cedead7d382e82993dca0959a5cfd37).